### PR TITLE
Fix issues caused by tf2 update

### PR DESCRIPTION
**Pull request by BenCat07**

### CODE REVIEW

File: include/helpers.hpp [hunk 1]
```diff
@@ -226,3 +226,84 @@ template <typename... Args> std::string format(const Args &... args)
...
+    }
+};
+
+inline WeaponData_t *GetWeaponData(IClientEntity *weapon)
+{
+    weapon_info info(weapon);
+    int WeaponData = info.weapon_data;
+    int WeaponMode = info.weapon_mode;
+    return (WeaponData_t *) (WeaponData + sizeof(WeaponData_t) * WeaponMode + 1784);
+}
```
Comment (TotallyNotElite) [inline include/helpers.hpp:309]: I don't think this belong here. Why don't you make a new header for this?

**Review by TotallyNotElite — CHANGES_REQUESTED** [review]

File: include/helpers.hpp (hunk 1 above)
Comment (BenCat07) [reply include/helpers.hpp:309]: Sure

File: include/common.hpp [hunk 2]
```diff
@@ -76,6 +76,7 @@
 #include <CNavFile.h>
 #include "HookTools.hpp"
 #include "bytepatch.hpp"
+#include "WeaponData.hpp"
```
Comment (TotallyNotElite) [inline include/common.hpp:79]: I REALLY don't think its necessary for every translation unit to have this

**Review by TotallyNotElite — CHANGES_REQUESTED** [review]

File: include/common.hpp (hunk 2 above)
Comment (BenCat07) [reply include/common.hpp:79]: Resolved

**Review by TotallyNotElite — APPROVED** [review]
Assuming it's tested

**Comment by BenCat07** [comment]
> Assuming it's tested

indeed

**Pull request merged**